package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "spellcheck": "mdspell --en-gb --report --ignore-acronyms --ignore-numbers --no-suggestions 'content/**/*.md'",
     "broken-link-local": "blcl --recursive --ordered --exclude-external ./public/",
-    "broken-link-all": "blcl --recursive --ordered ./public/",
+    "broken-link-all": "blcl --recursive --exclude https://github.com/sourcebots/docs/tree/master/content/* --ordered ./public/",
```

Comment:
Please could we have an inline comment explaining the exclusion.

Meta: this will likely conflict with #117.

Reply:
Please see above note about @RealOrangeOne's comments in 1add520

Reply:
I understand the reason for the exclusion; I'm asking for a comment so that it's clear to all future readers of the code.

Please bear in mind that future readers _won't_ have to hand the comments on this PR or the original commit which removed the functionality. While they _could_ `git blame`, the chances are that they'd have to chase several layers of commits to find the rationale (and then they'd need to work backwards from "Add Github Edit Links" to this PR and onwards from there). Adding a comment now is easy and makes things clearer for everyone.

Reply:
Ah, I missed the word inline and mistook the context of your request.

Fixed in 9d037ff